build.sbt

```diff
@@ -37,10 +38,36 @@ lazy val microservice = Project(appName, file("."))
     // ***************
   )
 
-lazy val testSettings: Seq[Def.Setting[_]] = Seq(
-  fork := true,
-  javaOptions ++= Seq(
-    "-Dconfig.resource=test.application.conf",
-    "-Dlogger.resource=logback-test.xml"
-  )
-)
+lazy val unitTestSettings =
+  inConfig(Test)(Defaults.testTasks) ++
+    Seq(
+      testOptions in Test := Seq(
+        Tests.Filters(
+          Seq(
+            _ startsWith "connectors",
```

Comment:
We should fix this, happy to pair on it - but the other things mentioned are higher priority.